quickstarts/kind/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Kubernes Kind Quickstart
+layout: default
+---
+
+# Installing the dependencies
+
+This quickstart assumes that you have the latest version of the `kind` binary, which you can get [here](https://github.com/kubernetes-sigs/kind/releases).
+
+Kind requires a running Docker Daemon. There are different Docker options depending on your host platform.
+You can follow the instructions [here](https://docs.docker.com/get-docker/).
+
+You'll also need `kubectl` binary, which you can get by following the instruction [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+Once you have all the binaries installed, and a Docker daemon running, make sure everything works:
+
+```shell
+# Validate docker installation
+docker ps
+docker version
+
+# Validate kind
+kind version
+
+# Validate kubectl
+kubectl version
+```
+
+# Configuring the Docker daemon
+
+If your Docker Daemon runs as a VM you'll most likely need to configure how much memory the VM should have, how many CPUs, how much disk space, and swap size.
+Make sure to assign at least 2 CPUs, and preferably 4 Gb or more of RAM. Consult Docker documentation for you platform.
+
+
+# Starting Kubernetes cluster
+
+This will start a local development cluster of [Kubernetes Kind](https://github.com/kubernetes-sigs/kind) which installs as a single docker container.
+
+```shell
+# Install kind
```

Comment:
I wonder if we need this comment really. We do not have it in the follow up commands. But in any case, if you keep it it should probably say _Create Kind cluster_ nd not _Install kind_?

Reply:
True. Will remove it.